css-pseudo-classes/style.css

```diff
@@ -0,0 +1,36 @@
+div {
+  text-align: center;
+}
+
+button {
+  padding: 0.5rem 2rem;
+  border-width: thin;
+  margin: 5px 5px;
+  background-color: white;
+  font-size: 1.25rem;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+  border-color: lightgray;
+  border-width: medium;
+}
+
+table {
+  margin-left: 665px;
+  margin-right: auto;
+}
+
+thead {
+  background-color: black;
+  color: white;
+}
+
+td {
+  text-align: left;
+}
+
+.table-color {
```

Comment:
while this does work, the goal of the exercise is to use the `nth-child` pseudo class. Let me know if you have any questions on using that!